src/@next/Modal/Modal.tsx

```diff
@@ -55,6 +58,8 @@ export const Modal = React.forwardRef<HTMLDivElement, ModalProps>(
       onClose,
       onBack,
       zIndexOverride = 999,
+      leftComponent,
```

Comment:
wondering if we can make it a bit more meaningful, ex: forward, backward icon/component